components/runtime/wine/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
```

Comment:
misprint in license header  ("CDDL)" instead of ("CDDL")

Reply:
fixed.
JFYI, there are many places with this misprint.
see the output of
ggrep -rF '("CDDL)"' components